### ISSUE TITLE
Antibiogram

### ISSUE THREAD

**Issue by MiekeNXH** [hl7-be/lab#141]
How can we recognise an Antibiogram? In the examples from Macsys a LOINC code is added which has the label antibiogram. Will all the labs do this? 
We also learned that labs can use preformatted text for sending an antibiogram. What is the added value of sending an antibiogram in preformatted text vs PDF?

**Comment by bdc-ehealth** [comment]
1) This approach seems reasonable to me. Should be confirmed by the WG.

2) Not all lab softwares are able to send Antibiograms in a structured way. Because we do not want to block the adoption of the structured format for the other tests, those labs can still send the antibiogram results as formatted text in the FHIR message. The contents of the PDF and the FHIR message must be equivalent, that's why the information must be in both formats.

**Comment by filipv-nh** [comment]
Is it the responsibility of the labsofts to make sure that the contents of the attached PDF and/or the formatted text in the FHIR message is equivalent in case they send both?
On the receiving side, we are not going to check this.

**Comment by bdc-ehealth** [comment]
Indeed, this was the result of the discussions we had about this in the workgroup. @alinderom will confirm this.

**Comment by alinderom** [comment]
I confirm
-  not all labs can send AB-gram in a structured way, so formatted text will be an alternative
- it's always the sender that has to see that the PDF and formatted FHIR text corresponds. The "independend" viewer can help the sender to check if he formetted well his message

**Comment by thibaultmh** [comment]
> 1. This approach seems reasonable to me. Should be confirmed by the WG.
> 
>     2. Not all lab softwares are able to send Antibiograms in a structured way. Because we do not want to block the adoption of the structured format for the other tests, those labs can still send the antibiogram results as formatted text in the FHIR message. The contents of the PDF and the FHIR message must be equivalent, that's why the information must be in both formats.

About 1., I suggest looking at the observation interpretation or value. If it's RIS you know it's an antibiogram I think? I see Macsys does it different than us but perhaps we could come to a consensus and add it to the IG?

**Comment by bdc-ehealth** [comment]
Question 1) the examples from MACSYS set the right standard for the indication of the Antibiogram.
Question 2) the contents of the PDF and the structured contents in the FHIR document should be identical. This is the responsability of the lab software.

**Comment by bdc-ehealth** [comment]
MIPS will check if the solution of an extra level indicating "antibiogram" and adding RIS, AGAR, E-test, MIC as sublevels is possible.

**Comment by pablocorilus** [comment]
@bdc-ehealth is there any decision on how to indicate that the observations are about a Antibiogram.
We are making the labo report, and for the moment we support two different methods, both by looking at the examples.
For one we the observation code = 29576-76 (loinc) and the other is observation code = ANTIBIOGRAM

**Comment by bdc-ehealth** [comment]
COW: Thibault proposes to add a tag in the meta section of the Observation, containing a general code for susceptibility tests/isolation tests (e.g. LOINC - to be clarified) to make clear this is an antibiogram. Alexis and Kristof will ask their client teams if this is a good way of working. The idea is to facilitate the receiver. Tom Tollenaere suggests that this information is already present because we can add the LOINC titles in FHIR cfr. HUB metadata. We need input of clinical biologists.

**Comment by pablocorilus** [comment]
@bdc-ehealth thank you for the follow up

**Comment by bdc-ehealth** [comment]
COW: we ask for some international advice @costateixeira . Another possibility would be to use the category field for this (where we could put the LOINC panel code).

**Comment by nicovn** [comment]
Colleagues in Germany have analysed microbiology in FHIR messaging very detailed. 

The work they've done looks very good !

Please check out their work at:

[https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10517923/](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10517923/)

Example messages:

[https://simplifier.net/Medizininformatik-Initiative-Modul-Mikrobiologie/~resources?category=Example&sortBy=RankScore_desc](https://simplifier.net/Medizininformatik-Initiative-Modul-Mikrobiologie/~resources?category=Example&sortBy=RankScore_desc)

FHIR specs

https://simplifier.net/guide/Copy-of---EN--Medizininformatik-Initiative---Modul-Mikrobiologie/IG-MII-Modul-Microbiology/Technical-Implementation/FHIR-Profile/Culture-diagnostics/Colony-count.page.md?version=current

Nico

**Comment by bdc-ehealth** [comment]
@nicovn Thanks, Nico, for the hint, we will certainly take it into account during the next meetings.
